rubyatscale/packs
# add use_pks

* [x] I bumped the gem version (or don't need to) 💎

## What's up 
pks uses `enforcement_globs_ignore` which bin/packs doesn't have. Now that the ZP code is using this we are failing `bin/packwerk update-todo`. 

This pr introduces `use_pks` to the config so that we can use `pks check` in ZP CI. 

## How to test 

1. In ZP (or whatever other repo is using packs) gemfile update path to point to your local version 
2. ` gem 'packs', require: false, path: '../packs' ` 
3. Add `config.use_pks = true` to `config/packs.rb` (line 9) 
4. run `bundle`
 (verify you're pointing to your local packs with `bundle show packs`)
5. run `bin/packs lint_package_todo_yml_files` 
6. 🎉  see success message!
![image](https://github.com/rubyatscale/packs/assets/16791208/5bcfece5-6345-44dc-9cdb-ba01799a1083)

<img width="740" alt="image" src="https://github.com/rubyatscale/packs/assets/16791208/0e931d1b-bf5b-464b-aad7-95044f98619e">



set `config.use_pks = false` and run the commands: 
(you can tell it's running packwerk by all the dots) 
bin/packs check -> fails
bin/packs validate -> fails 
bin/packs update -> fails
